solidity/dashboard/src/store.js

```diff
@@ -2,10 +2,13 @@ import { createStore, applyMiddleware } from "redux"
 import createSagaMiddleware from "redux-saga"
 import rootReducer from "./reducers"
 import rootSaga from "./sagas"
-
+import { composeWithDevTools } from "redux-devtools-extension"
```

Comment:
I think we don't need the redux devtools on prod so maybe we could do
```
import { composeWithDevTools } from "redux-devtools-extension/developmentOnly"
```

as stated [here](https://www.npmjs.com/package/redux-devtools-extension)?